api.js

```diff
@@ -154,7 +154,7 @@ exports.init = function (sbot, config) {
     // Pluck relevant internal APIs
     const { deriveRootMetaFeedKeyFromSeed } = sbot.metafeeds.keys
     const { getSeed, getAnnounces, getLatest } = sbot.metafeeds.query
-    const { generateSeedSaveMsg, generateAnnounceMsg, addExistingFeed } =
+    const { contentSeed, contentAnnounce, msgValAddExisting } =
```

Comment:
Only minor thing I have is that in this file it gets a bit harder to read maybe what contentSeed does because we extract the functions from the object. Like the tests are `messages.contentSeed` which makes it a bit easier to see that is has something to with generating messages. One way to maybe improve this would be to have just messages shorted here, so the lines in this file becomes the same as in the tests. What do you think?

Reply:
Ah yes, I see what you mean. I can do what you said, but how about (new idea) we just prefix all these functions with `get`?

Reply:
also works, probably a better idea